app/policies/message_policy.rb

```diff
@@ -0,0 +1,9 @@
+class MessagePolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def create?
+    true
```

Comment:
I said in a lot of pr before, I don't like this policy rule, at least try to use user.present? to check that is an authenticated user.

Reply:
Yep i update this policy.